ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/forkchoice/VoteTracker.java

```diff
@@ -26,11 +26,25 @@ public class VoteTracker {
   private final Bytes32 currentRoot;
   private final Bytes32 nextRoot;
   private final UInt64 nextEpoch;
+  private final boolean equivocated;
```

Comment:
Votes are stored on disk so we'll need to ensure we handle migration correctly.

Reply:
Actually we don't seem to have updated the storage of votes on disk at all and will probably need to - otherwise we'll reapply weightings we previously removed on a restart. `VoteTrackerSerializer` has the code for this. It should be easy enough to read data with or without the extra boolean by checking if we've reached the end of the SSZ stream before reading it.

We need to avoid writing out the new boolean when the feature is toggled off.

Reply:
Got you